lib/features/taskcluster_proxy.js

```diff
@@ -84,6 +85,34 @@ export default class TaskclusterProxy {
       throw new Error('Failed to initialize taskcluster proxy service.');
     }
 
+    // Update credentials in proxy
+    task.on('credentials', async (credentials) => {
+      let credentials = JSON.stringify(task.claim.credentials);
+
+      await new Promise((accept, reject) => {
+        let req = http.request({
+          hostname: inspect.NetworkSettings.IPAddress,
+          method: 'PUT',
+          path: '/credentials',
+          headers: {
+            'Content-Type': 'text/json',
+            'Content-Length': credentials.length
+          }
+        }, res => {
+          if (res.statusCode == 200) {
+            task.runtime.log('Credentials updated');
+            accept();
+          } else {
+            reject(new Error(`Credentials update failed ${res.statusCode}`));
```

Comment:
before rejecting, could we log this failure like we do above for when it's successful?